### PR TITLE
Feature/2 create mock data

### PR DESCRIPTION
**Pull request by NerevitVasyan**

### CODE REVIEW

File: BobManager.API/BobManager.API/Program.cs [hunk 1]
```diff
@@ -3,18 +3,44 @@
...
+                catch (Exception ex)
+                {
+                    throw ex;
+                }
+            }
+            return webHost;
+        }
+    }
+
+
```
Comment (ander21) [inline BobManager.API/BobManager.API/Program.cs:38]: Remove empty line.

File: BobManager.API/BobManager.DataAccess/Configuration/EntityInitializer.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+            await InitializeIdetity();
+            foreach(var initializer in typeInitializers)
+            {
+                await initializer.Init(context);
+                await context.SaveChangesAsync();
+            }
+        }
+
+        private async Task InitializeIdetity()
```
Comment (ander21) [inline BobManager.API/BobManager.DataAccess/Configuration/EntityInitializer.cs:57]: You miss the letter on the method name.

File: BobManager.API/BobManager.DataAccess/Configuration/TypeConfigurations/GroupRoleInit.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        public async Task Init(DbContext context)
+        {
+            GroupRole[] groupRoles = new GroupRole[]
+            {
+                new GroupRole { Name = "Lol"},
+                new GroupRole { Name = "Kek"},
+                new GroupRole { Name = "Cheburek"},
+                new GroupRole { Name = "Garold"},
+                new GroupRole { Name = "Mona Liza" }
+
```
Comment (ander21) [inline BobManager.API/BobManager.DataAccess/Configuration/TypeConfigurations/GroupRoleInit.cs:22]: Empty line

**Review by Firto — APPROVED** [review]
Approve

**Review by ander21 — APPROVED** [review]
Approve with comments

**Review by Liosh-a — APPROVED** [review]
Approve

**Pull request merged**